wolox-books-vue/src/views/NavBar.vue

```diff
@@ -0,0 +1,40 @@
+<template lang="pug">
+  nav.navbar.row.middle.space-around.full-width
+    div.main.row.space-between
```

Comment:
remove divs

Reply:
the div is to have the logo and the logout button inside the inner container